pandas/io/pytables.py

```diff
@@ -1452,11 +1452,10 @@ def error(t):
                         "nor a value are passed"
                     )
             else:
-
                 try:
-                    pt = _TYPE_MAP[type(value)]
+                    pt = {Series: "series", DataFrame: "frame"}[type(value)]
                 except KeyError:
-                    error("_TYPE_MAP")
+                    raise error("_TYPE_MAP")
```

Comment:
will this make sense now that _TYPE_MAP is inlined?

Reply:
I guess we could inline the thing here too